generators/client/templates/webpack/webpack.common.js

```diff
@@ -96,5 +91,22 @@ module.exports = {
         net: 'empty',
         tls: 'empty',
         child_process: 'empty'
-    }
+    },
+    plugins: [
+        new VueLoaderPlugin(),
+        new CopyWebpackPlugin([
+            // { from: './node_modules/swagger-ui/dist/css', to: 'swagger-ui/dist/css' },
+            // { from: './node_modules/swagger-ui/dist/lib', to: 'swagger-ui/dist/lib' },
+            // { from: './node_modules/swagger-ui/dist/swagger-ui.min.js', to: 'swagger-ui/dist/swagger-ui.min.js' },
+            { from: './src/main/webapp/swagger-ui/', to: 'swagger-ui' },
```

Comment:
I don't know if it can help, but the Travis logs are:
```
[INFO] ERROR in [copy-webpack-plugin] unable to locate './src/main/webapp/swagger-ui/' at '/home/travis/app/src/main/webapp/swagger-ui'
[INFO] 
[INFO] ERROR in [copy-webpack-plugin] unable to locate './src/main/webapp/favicon.ico' at '/home/travis/app/src/main/webapp/favicon.ico'
[INFO] 
[INFO] ERROR in [copy-webpack-plugin] unable to locate './src/main/webapp/manifest.webapp' at '/home/travis/app/src/main/webapp/manifest.webapp'
[INFO] 
[INFO] ERROR in [copy-webpack-plugin] unable to locate './src/main/webapp/robots.txt' at '/home/travis/app/src/main/webapp/robots.txt'
```

Reply:
Thanks, seems that we don't yet have swagger-ui in vue :( I'll check that on Sunday.